addons/website_mass_mailing/controllers/website_form.py

```diff
@@ -12,7 +12,10 @@ class WebsiteNewsletterForm(WebsiteForm):
 
     def _handle_website_form(self, model_name, **kwargs):
         if model_name == 'mailing.contact':
-            list_ids = [int(x) for x in kwargs['list_ids'].split(',')]
+            list_ids = kwargs.get('list_ids', None)
+            if not list_ids:
+                return json.dumps({'error': _('Mailing List(s) not found!')})
```

Comment:
Whenever you are adding new terms in stable, you need to add those to the `.pot` file.
Don't forget to insert it into the `.pot` file **exactly** where it would be if you exported the translation through Odoo UI: it will help a lot when comparing exported file and local file later should someone work with those file.
For what it's worth, here is a summary of how it works if you are not familiar with: https://github.com/odoo/odoo/pull/113570#pullrequestreview-1313445887